modular_citadel/code/modules/projectiles/guns/pumpenergy.dm

```diff
@@ -200,8 +206,38 @@
 	color = null
 	nodamage = 1
 	knockdown = 100
-	stamina = 5
+	stamina = 25
```

Comment:
There was *very* good reason why this was reduced to 5 stam; the particle defender's electrode mode is meant primarily for riot control, to get a horde of greytiders down quickly, but not too strong to the point where it surpasses the taser in terms of keeping singular targets down.